packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -98,9 +100,22 @@ export class SDKClient {
   private operatorAccountGauge;
   private maxChunks;
   private fileAppendChunkSize;
+  private mirrorNodeClient;
+
+  private readonly MirrorNodeRetries = parseNumericEnvVar(
+    'MIRROR_NODE_GET_CONTRACT_RESULTS_RETRIES',
+    'MIRROR_NODE_GET_CONTRACT_RESULTS_DEFAULT_RETRIES',
+  );
```

Comment:
This sounds like something that's a responsibility of the `MirrorNodeClient`, let's move it there.